packages/block-editor/src/components/inserter/preview-panel.js

```diff
@@ -15,8 +16,11 @@ import BlockCard from '../block-card';
 import BlockPreview from '../block-preview';
 
 function InserterPreviewPanel( { item } ) {
-	const { name, title, icon, description, initialAttributes, example } = item;
+	const { name, title, icon, description, initialAttributes } = item;
 	const isReusable = isReusableBlock( item );
+	const hoveredItemBlockType = getBlockType( name );
```

Comment:
Not a block, and I don't know if it even makes a huge difference, but it might be possible to avoid calling `getBlockType` if we check if `example` is set.

I was trying to do something like that over in https://github.com/WordPress/gutenberg/pull/43883/files#diff-2f239aff0c373f78c87b3d4930b1fd79277d01ee9a8ab407694fd5c4925d148fR21